tools/LogCollector/source/omslinux_agentlog.py

```diff
@@ -456,6 +456,66 @@ def copyExtensionFiles():
     writeLogCommand(cmd)
     return 0
 
+'''
+Copy Update Management Solution logs into /tmp/updatelogs
+'''
+def copyUpdateFiles():
+    cmd='mkdir -p /tmp/omslogs/updateMgmtlogs'
+    out=execCommand(cmd)
+    writeLogCommand(cmd)
+
+    cmd='cp /var/opt/microsoft/omsagent/log/urp.log /tmp/omslogs/updateMgmtlogs'
+    out=execCommand2(cmd)
```

Comment:
here you are using execCommand2, just want to be sure that this is the desired behavior.

Reply:
good eye!, fixed it ,did not impact as one returned the return code and other output for a cp command, so went unnoticed.